openedx/features/course_experience/api/v1/views.py

```diff
@@ -102,9 +110,9 @@ def reset_course_deadlines(request):
             'link_text': _('View all dates'),
             'message': _('Deadlines successfully reset.'),
         })
-    except Exception as e:
-        log.exception(e)
-        raise UnableToResetDeadlines  # lint-amnesty, pylint: disable=raise-missing-from
+    except Exception as reset_deadlines_exception:
+        log.exception(reset_deadlines_exception)
```

Comment:
This isn't your fault, but while you're there...  I don't believe this is how you use this method. The exception is grabbed behind the scenes by `log.exception` and you instead give it a nice error message, which is presented alongside the exception string/traceback. Search for other uses of it, as an example.

Something like `log.exception('Could not reset deadlines, sorry bruh')`

(But nice amnesty cleanup)